docs/config.md

```diff
@@ -0,0 +1,266 @@
+
+### requiredPropsFirst
+
+Shows required properties in schemas first, ordered in the same order as in the required array.
+
+### schemaExpansionLevel
+
+Specifies whether to automatically expand schemas in Reference docs. Set it to `all` to expand all schemas regardless of their level, or set it to a number to expand schemas up to the specified level. For example, `schemaExpansionLevel: 3` expands schemas up to three levels deep. The default value is `0`, meaning no schemas are expanded automatically.
+
+### scrollYOffset
+
+Specifies a vertical scroll-offset.
+This setting is useful when there are fixed positioned elements at the top of the page, such as navbars, headers, etc.
+
+Note that you can specify the `scrollYOffset` value in any of the following ways:
+- as a number - a fixed number of pixels to be used as the offset.
+- as a CSS selector - the selector of the element to be used for specifying the offset. The distance from the top of the page to the element's bottom is used as the offset.
+- a function (advanced) - a getter function. Must return a number representing the offset (in pixels).
+
+### showExtensions
+
+Shows specification extensions ('x-' fields). Extensions used by Redoc are ignored. The value can be boolean or an array of strings with names of extensions to display. When used as boolean and set to `true`, all specification extensions are shown.
+
+### showObjectSchemaExamples
+
+Shows object schema example in the properties; default `false`.
+
+### showWebhookVerb
+
+When set to `true`, shows the HTTP request method for webhooks in operations and in the sidebar.
+
+### simpleOneOfTypeLabel
+
+Shows only unique `oneOf` types in the label without titles.
+
+### sortEnumValuesAlphabetically
+
+When set to `true`, sorts all enum values in all schemas alphabetically.
+
+### sortOperationsAlphabetically
+
+When set to `true`, sorts operations in the navigation sidebar and in the middle panel alphabetically.
+
+### sortPropsAlphabetically
+
+When set to `true`, sorts properties in all schemas alphabetically.
+
+### sortTagsAlphabetically
+
+When set to true, sorts tags in the navigation sidebar and in the middle panel alphabetically. Note that only tags are sorted alphabetically in the middle panel, not the operations associated with each tag. To sort operations alphabetically as well, you must set the `sortOperationsAlphabetically` setting to `true`.
+
+_Default: false_
+
+### untrustedDefinition
+
+If set to `true`, the API definition is considered untrusted and all HTML/Markdown is sanitized to prevent XSS.
+
+## Theme settings
+
+* `spacing`
+  * `unit`: 5 # main spacing unit used in autocomputed theme values later
+  * `sectionHorizontal`: 40 # Horizontal section padding. COMPUTED: spacing.unit * 8
+  * `sectionVertical`: 40 # Horizontal section padding. COMPUTED: spacing.unit * 8
+* `breakpoints` # breakpoints for switching three/two and mobile view layouts
```

Comment:
```suggestion
* `breakpoints`: # breakpoints for switching three/two and mobile view layouts
```

Reply:
None of the other top-level entries have colons, so I'm not adding this one.